### PR TITLE
mediaimpact: startRender API

### PR DESCRIPTION
**Pull request by adtechnology**
added loadHandler functionality to support startRender API
 changed lib file url
changed prefetch and extended preconnect settings

### CODE REVIEW

**Comment by lannka** [comment]
@adtechnology could you fix the linter error so I can merge for you?

**Comment by lannka** [comment]
@adtechnology some more linter errors :-)

FYI, you can run lint locally: `gulp lint`

**Comment by adtechnology** [comment]
F**K, it was linted already, but I pasted your line from #5333 and not linted afterwards...

**Comment by lannka** [comment]
@adtechnology  there are still type check errors. Could you fix?
`gulp check-types`

File: ads/mediaimpact.js [hunk 1]
```diff
@@ -25,6 +25,21 @@ import {loadScript} from '../3p/3p';
 
 export function mediaimpact(global, data) {
   global.fif = false;
+  /* eslint google-camelcase/google-camelcase: 0 */
+  global.sas_loadHandler = function(f) {
+    if (f.hasAd) {
+      f.crea || (f.crea1 = {
```
Comment (lannka) [inline ads/mediaimpact.js]: just want to make sure this is not a typo here. should it be `f.crea` or `f.crea1`?

Comment (adtechnology) [inline ads/mediaimpact.js]: Yeah, right, THX again...

**Pull request merged**